sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_sas_builder.hpp

```diff
@@ -150,6 +155,11 @@ namespace Azure { namespace Storage { namespace Sas {
      */
     DeleteVersion = 64,
 
+    /**
+     * @brief Indicates that setting immutability policy is permitted.
+     */
+    SetImmutabilityPolicy = 128,
```

Comment:
Should we try to make the values of the `SetImmutabilityPolicy` flag enum be identical? I guess it might be confusing either way, but wanted to ask nonetheless.

Reply:
Well, I don't think that's necessary. Because customers should only use the enumerator's name, rather than value, and the value may change as version upgrades.